app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+    # binding.pry
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t "success_signup"
+      redirect_to("/users/#{@user.id}")
```

Comment:
dùng path đã define trong route đi em
```suggestion
      redirect_to @user
      # hoặc
      redirect_to user_path(@user.id)
```